templates/index.html

```diff
@@ -37,29 +26,559 @@ <h1>Take control of <br /><span style="display: block; height: 1.2em;" class="ty
+          </div>
+          <div class="col-3">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/52d0f6ab-traefik-logo-jujuis.png",
+                alt="",
+                height="288",
+                width="216",
+                hi_def=True,
+                loading="auto",
+                attrs={
+                  "class": "homepage-tabs-logo",
+                },
+              ) | safe
+            }}
+            <h3 class="p-heading--5">
+              <a href="https://charmhub.io/traefik-k8s">Traefik</a>
+            </h3>
+            <p>Open-source reverse proxy and load balancer for HTTP and TCP-based applications</p>
+          </div>
+          <div class="col-3">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/6320c190-landscape-logo-jujuis.png",
+                alt="",
+                height="288",
+                width="154",
+                hi_def=True,
+                loading="auto",
+                attrs={
+                  "class": "homepage-tabs-logo",
+                },
+              ) | safe
+            }}
+            <h3 class="p-heading--5">
+              <a href="https://charmhub.io/landscape-scalable">Landscape</a>
+            </h3>
+            <p>Ubuntu systems management tool developed by Canonical</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-section">
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <div class="u-fixed-width"><hr class="p-rule"></div>
+        <p>Learn more about charms on <a href="http://discourse.charmhub.io">Discourse</a> and <a href="http://chat.charmhub.io">Mattermost</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width"><hr class="p-rule"></div>
+    <div class="row">
+      <div class="col-7">
+        <h2>Watch: Learn about Juju in 10 minutes</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-6 col-start-large-4">
+        <iframe class="responsive-video" src="https://www.youtube.com/embed/HhRBNs9ckIk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        <p>Watch this video to learn the core concept of Juju and charms, how you can use  them to automate operations across different clouds/VM/k8s and the benefits of integrations.</p>
+      </div>
+    </div>
+  </section>
 
-  {% include 'partials/_what-is-juju.html' %}
+  <section class="p-section">
+    <div class="u-fixed-width"><hr class="p-rule"></div>
+    <div class="row">
+      <div class="col-7">
+        <h2>
+          <a href="/docs/juju/get-started-with-juju">Get started with Juju&nbsp;&rsaquo;</a>
+        </h2>
+      </div>
+    </div>
+  </section>
```

Comment:
```suggestion
```

I think we could cut this -- it looks a bit strange with no copy under the heading, and we already have the button at the top? WDYT?